.github/workflows/build.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v4
         with:
-          java-version: '17'
+          java-version: '21'
           distribution: 'temurin'
```

Comment:
possible to use `registry.access.redhat.com/ubi9/openjdk-21-runtime` (or `ubi8`)?

Reply:
Can be done differently

Reply:
I see your point to use the very same distribution in all places. But I do not think that using the ubi is possible here. This Java distribution in GitHub actions is for running the Maven build to compile and run the tests, only.